### ISSUE TITLE
Component - MCQ - Missing fieldset [RDI 2.35]

### ISSUE THREAD

**Issue by MeD-Owner** [MeD-DMC/Adapt-accessibilityfixes#75]
Using IE11, when a screen reader tabs through the page and lands on an answer the question is not announced with the answer. This also applies to the submit button for each question. 

Using Edge and Chrome, JAWS will announce the question with the first answer, but not for the other answers which we navigate to with the arrow keys. 

We do have concerns that the screen reader should also read out the question with each radio button option as we navigate with the arrow keys, but this requires more research. This would probably require modifications in the background coding of the component. 

- Our WAI-ARI fix should have worked. since this is in ie11 we could ignore it for now

**Comment by MeD-Owner** [comment]
Changed our original fix to instead add a fieldset and legend rather than WAI-ARIA

Will need to adapt the fix to graphical MCQ as well, maybe other interactions?

**Comment by MeD-Owner** [comment]
Fieldset and legend are doubled (see page 4 q2)

**Comment by MeD-Owner** [comment]
Fixed fieldset and legend doubling issue, should technically fix problems with label being associated with inputs

- to have Melissa re-test
